### PR TITLE
doxygen fixes

### PR DESCRIPTION
**Pull request by AlexanderSaydakov**

### CODE REVIEW

File: tuple/include/array_tuple_a_not_b.hpp [hunk 1]
```diff
@@ -28,14 +28,27 @@
...
+   * @param allocator to use for allocating and deallocating memory
+   */
   explicit array_tuple_a_not_b(uint64_t seed = DEFAULT_SEED, const Allocator& allocator = Allocator());
 
+  /**
+   * Computes the a-not-b set operation given two sketches.
+   * @param a sketch A
+   * @param b sketch B
+   * @param ordered optional flag to specify if ordered sketch should be produced
+   * @return the result of a-not-b as a compact sketch
```
Comment (jmalkin) [inline tuple/include/array_tuple_a_not_b.hpp:50]: Maybe A-not-B is more consistent with the input sketch names?

Comment (jmalkin) [reply tuple/include/array_tuple_a_not_b.hpp:50]: Nevermind. Variables are lowercase. Maybe call them a and b in the above @param lines?

File: tuple/include/array_tuple_sketch.hpp [hunk 2]
```diff
@@ -116,6 +111,10 @@ template<typename Array, typename Allocator> class compact_array_tuple_sketch;
 
 /**
  * Update array tuple sketch.
+ * This is a wrapper around tuple sketch to match the functionality and serialization format of ArrayOfDoublesSketch in Java.
+ * For this the sketch must be configured with array<double> or std::vector<double>.
+ * This is a more generic implementation for any integral type (serialization assumes contiguous array size_of(T) * num_values).
```
Comment (jmalkin) [inline tuple/include/array_tuple_sketch.hpp:116]: Numeric type, not integral

File: tuple/include/tuple_a_not_b.hpp [hunk 3]
```diff
@@ -37,11 +37,19 @@ class tuple_a_not_b {
...
+   * Constructor
+   * @param seed for the hash function that was used to create the sketch
+   * @param allocator to use for allocating and deallocating memory
+   */
   explicit tuple_a_not_b(uint64_t seed = DEFAULT_SEED, const Allocator& allocator = Allocator());
 
   /**
    * Computes the a-not-b set operation given two sketches.
-   * @return the result of a-not-b
+   * @param a sketch A
```
Comment (jmalkin) [inline tuple/include/tuple_a_not_b.hpp:49]: Same as above with A vs a but probably don't bother changing either place.

File: tuple/include/array_tuple_sketch.hpp (hunk 2 above)
Comment (AlexanderSaydakov) [reply tuple/include/array_tuple_sketch.hpp:116]: right, integral does not include floating point. arithmetic, perhaps

**Review by jmalkin — APPROVED** [review]
Looks good

**Pull request merged**